MAUI/Effects-View/Effects/Scale.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: The .NET MAUI Effects View control | Syncfusion
```

Comment:
The .NET MAUI Scale Animation Effects | Effects View control | Syncfusion

Reply:
Updated.